web/package.json

```diff
@@ -42,8 +53,13 @@
     "karma-jasmine": "~3.3.0",
     "karma-jasmine-html-reporter": "^1.5.0",
     "protractor": "~7.0.0",
+    "protractor-image-comparison": "^3.9.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
Do we need these? I might be wrong but I believe we're only dealing with Angular/TypeScript and nothing with React

Reply:
I had to download those for happo to run.